classes/sdcard_image-rpi.bbclass

```diff
@@ -148,6 +149,17 @@ IMAGE_CMD_rpi-sdimg () {
         fi
     fi
 
+    # Add files (eg. hypervisor binaries) from the deploy dir
+    if [ -n "${DEPLOYPAYLOAD}" ] ; then
+        echo "Copying deploy file payload into VFAT"
+        for entry in ${DEPLOYPAYLOAD} ; do
+            # Split the entry at ':' to enable file renaming for the destination
+            DEPLOY_FILE="$(echo $entry | cut -f1 -d:)"
+            DEST_FILENAME="$(echo $entry | cut -f2 -d:)"
```

Comment:
Would make sense to default DEST_FILENAME to DEST_FILENAME when not provided? So functionally `example2:example2` would be the same as `example2`.

Reply:
ack - have just refreshed the branch with that implemented and documented